tests/py/test_security.py

```diff
@@ -2,10 +2,13 @@
 from pytest import yield_fixture
 
 from aspen.testing.client import Client
+from inside_gratipay.main import website
 
 @yield_fixture
 def client():
-    yield Client(www_root='www', project_root='')
+    client = Client(www_root='www', project_root='.')
+    client._website = website
```

Comment:
We're telling Aspen's testing infrastructure to use the `website` that we are making over in `inside_gratipay/main.py`, which is configured how we want it.

Under previous versions of Aspen, Aspen would look in the project root for a magically named file ("`configure-aspen.py`") and take configuration from there. The magic is gone in 0.42.